### PR TITLE
Resolve jQuery isFunction warning

### PR DESCRIPTION
**Pull request by redJ4y**
[Issue raised by Mattybat on the Raygun forums.](https://raygun.com/forums/thread/143932)

See https://api.jquery.com/jQuery.isFunction/
> As of jQuery 3.3, `jQuery.isFunction()` has been deprecated. In most cases, its use can be replaced by `typeof x === "function"`.

---

This PR also
- allows Continuous Integration action to run without manually setting the chromedriver version in `package.json`,
- revives `prerelease.yml`.

### CODE REVIEW

**Review by Hamish-taylor — COMMENTED** [review]
I am happy with these changes, but I will leave it to JD or Panos for the final approval

File: src/raygun.tracekit.jquery.js [hunk 1]
```diff
@@ -55,7 +55,7 @@
 
     var keys = ['complete', 'error', 'success'], key;
     while(key = keys.pop()) {
-      if ($.isFunction(options[key])) {
+      if (typeof options[key] === "function") {
```
Comment (redJ4y) [inline src/raygun.tracekit.jquery.js:58]: This is the only change to the provider. The rest is related to Continuous Integration tests.

**Review by PanosNB — APPROVED** [review]
Looks good to me

**Review by PanosNB — APPROVED** [review]
Good work automating this Jared!

**Pull request merged**